cli/command/context/export.go

```diff
@@ -77,7 +77,7 @@ func writeTo(dockerCli command.Cli, reader io.Reader, dest string) error {
 
 // RunExport exports a Docker context
 func RunExport(dockerCli command.Cli, opts *ExportOptions) error {
-	if err := validateContextName(opts.ContextName); err != nil {
+	if err := validateContextName(opts.ContextName); err != nil && opts.ContextName != "default" {
```

Comment:
Should probably use the `const` here

```suggestion
	if err := validateContextName(opts.ContextName); err != nil && opts.ContextName != command.DefaultContextName {
```

Also wondering if this exception is still needed, if `ContextStoreWithDefault ` implements the interface (so `GetContextMetadata` would work?)